_posts/static/2018-02-01-installation-wizard.md

```diff
@@ -80,10 +80,10 @@ Flyve MDM v2.0.0 runs on GLPI 9.2.1 or later. It depends on inventory features o
   * Using Git, for those who are familiar with it, be aware of the version:
 
     ```console
-      git clone https://github.com/fusioninventory/fusioninventory-for-glpi.git
+      git clone https://github.com/fusioninventory/fusioninventory-for-glpi.git fusioninventory
     ```
 * Go to glpi/plugins/ and unpack the file there
-* Rename the folder to ```fusioninventory```
+* Rename the folder to ```fusioninventory``` if needed.
```

Comment:
renaming the folder becomes useless if the plugin is deployed with GIT.

Reply:
Done, I removed the rename steps and moved the glpi/plugins location before starting the downloads